README.md

```diff
@@ -66,6 +66,7 @@ Lightning Terminal is backwards compatible with `lnd` back to version v0.11.1-be
 
 | LiT              | LND          |
 | ---------------- | ------------ |
+| **v0.3.5-alpha** | v0.11.1-beta | 
```

Comment:
Shouldn't this be v0.12.0-beta like below?

Reply:
In theory we should still be backward compatible down to `v0.11.1-beta`, at least for remote connections.

Reply:
Doh! I should've probably read the sentence above that wasn't visible in the diff. Ok, that makes sense.